src/onegov/org/models/ticket.py

```diff
@@ -67,6 +71,9 @@ def reference(self, request: 'OrgRequest') -> str:
     def reference_group(self, request: 'OrgRequest') -> str:
         return request.translate(self.group)
 
+    # FIXME: extra localized text is used for suggestions in the search but
+    # does not work with the ranking of the search results
+    # @hybrid_property
     @cached_property
     def extra_localized_text(self) -> str:
```

Comment:
`extra_localized_text` is used for search suggestions but do not work when applying the ranking for search results. I changed to `hybrid_property` which throws
```
  File "/home/reto/workspace/onegov-cloud/src/onegov/org/models/ticket.py", line 93, in extra_localized_text
    q = object_session(self).query(Message)
        ^^^^^^^^^^^^^^^^^^^^
  File "/home/reto/workspace/onegov-cloud/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 3464, in object_session
    util.raise_(
  File "/home/reto/workspace/onegov-cloud/venv/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
sqlalchemy.orm.exc.UnmappedInstanceError: Class 'sqlalchemy.ext.declarative.api.DeclarativeMeta' is not mapped; was a class (onegov.org.models.ticket.FormSubmissionTicket) supplied where an instance was required?
```
How can I transform it to a hybrid property?